server/api/controllers/users/router.ts

```diff
@@ -4,4 +4,5 @@ export default express
   .Router()
   .post("/", controller.create)
   .get("/", controller.getAll)
-  .get("/:id", controller.getById);
+  .get("/:id", controller.getById)
```

Comment:
@Asdesai99 in your `PUT` request you used `user_id` instead of `id`. I think it would be good to keep it consistent so maybe changing `id` to `user_id` here would be a good idea. What do you think?

Reply:
nice catch I will make that change, thanks!